packages/fluentui/react-builder/src/components/DropSelector.tsx

```diff
@@ -105,6 +104,12 @@ export const DropSelector: React.FunctionComponent<DropSelectorProps> = ({
       selectorRef.current.style.width = `${targetRect.width}px`;
       selectorRef.current.style.height = `${targetRect.height}px`;
 
+      if (!isValidDrop(jsonTreeElement)) {
```

Comment:
When dropping close to the edge, the component will not be dropped inside the hovered component, but next to it.
In that case the background should not be red:
![image](https://user-images.githubusercontent.com/9615899/93194757-bdaa9300-f748-11ea-8fec-268d107e60f5.png)